src/main/java/com/google/sps/servlets/AddSchoolInfo.java

```diff
@@ -0,0 +1,159 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/add-info")
+public class AddSchoolInfo extends HttpServlet {
+
+  private Key existingSchoolKey;
+  private Key existingCourseKey;
+  private Key existingProfessorKey;
+
+  private Key newSchoolKey;
+  private Key newCourseKey;
+  private Key newProfessorKey;
+  private Key newTermKey;
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    String schoolName = request.getParameter("school-name");
+    String courseName = request.getParameter("course-name");
+    String term = request.getParameter("term");
+    Integer units = Integer.parseInt(request.getParameter("units"));
```

Comment:
Don't you have to say "int" as type instead of "Integer"? (I remember that I tried to run with Integer and for some reason told me it did not exist)

Reply:
I have actually always been using Integer this entire internship lol it seems to work

Reply:
JAJAJA perfect, good that it works for youu :)